tests/Toolkit/StrTest.php

```diff
@@ -425,6 +425,39 @@ public function testFromWithEmptyNeedle()
         Str::from('test', '');
     }
 
+    /**
+     * @covers ::increment
+     */
+    public function testIncrement()
```

Comment:
Could you please also add test cases for the following:

- Input string already contains numbers, but with a different separator; in these cases the method should add the separator and the `$first` number to the end and keep the existing number
- Two cases that show that the `$first` argument is ignored when there's already a number in the input. One case e.g. with an input of `Pöst_5` and `$first = 10` and one with `Pöst_15` and `$first = 10`.